src/Client/Web/Bit.Client.Web.BlazorUI/Components/Buttons/BitToggleButton.razor.cs

```diff
@@ -42,6 +42,9 @@ public bool IsChecked
             }
         }
         
+        /// <summary>
+        /// Callback called when IsChecked is changed.
+        /// </summary>
         [Parameter] public EventCallback<bool> IsCheckedChanged { get; set; }
```

Comment:
please remove this documentation comment.
it is not correct for these specific parameters.